.vib/appsmith/cypress/cypress/integration/appsmith_spec.js

```diff
@@ -14,10 +14,11 @@ it('allows to create a new project', () => {
   cy.get('[class*="templates-tab"]').click();
 
   // Create an application from the Marketing Portal template
-  cy.contains('[data-cy="template-card"]', 'Marketing Portal').within(() => {
-    cy.get('[class*="fork-button"]').click();
-  })
-  cy.contains('FORK TEMPLATE').click();
+  cy.get('input[data-testid*="search-input"]').type('Marketing Portal');
+  // Wait for only the Marketplace template to show in the search
+  cy.wait(5000);
```

Comment:
Please try not to use `cy.wait, this is considered an [anti-pattern](https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting) by Cypress and it doesn't assure you waiting 5 seconds is going to work every time.